std/internal/math/biguintcore.d

```diff
@@ -706,6 +706,29 @@ public:
         return BigUint(removeLeadingZeros(trustedAssumeUnique(rem)));
     }
 
+    // Return x / y in quotient, x % y in remainder
```

Comment:
Should add a warning comment that it doesn't handle signs...

Reply:
Or maybe it's better to simply move this logic into this function, so that future callers don't get surprised?

Reply:
I don't think anyone expects a function that works on `BigUint`s to handle signs.